test/test_autoload.py

```diff
@@ -0,0 +1,23 @@
+# Owner(s): ["module: unknown"]
+
+import os
+
+import torch  # noqa: F401
```

Comment:
Why do you need this? The import of torch.testing below will do this by side effect.